server-utilities/src/main/java/org/jboss/eap/qe/microprofile/common/utilities/WildFlyServerHelper.java

```diff
@@ -0,0 +1,25 @@
+package org.jboss.eap.qe.microprofile.common.utilities;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Class aggregating utilities for server
+ */
+public final class WildFlyServerHelper {
+
+    private WildFlyServerHelper() {
+        //intentionally left empty
+    }
+
+    /**
+     * Return path to server's log file
+     * @return path to log file
+     */
+    public static File getPathToLogFile() {
```

Comment:
- log path can be adjusted - https://docs.wildfly.org/17/Admin_Guide.html#paths
 - log path for standalone server
 - getPath.... and File is returned

method name should reflect "reality", what about `public static Path getDefaultStandaloneServerLog()`


You could possibly also introduce more robust method which is resolving `jboss.server.log.dir` from running server (would need Creaper dependency).
I guess something for the future, please create issue for that.
Here is my CLI experiment:
```
[standalone@localhost:9990 /] /:resolve-expression(expression="${jboss.server.log.dir}")
{
    "outcome" => "success",
    "result" => "/Users/rsvoboda/TESTING/wildfly-16.0.0.Beta1-SNAPSHOT/standalone/log"
}
```